tailwind.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        hackathon: {
```

Comment:
wait the hackathon colors should still be here cause all the portals will have no colors